Makefile

```diff
@@ -0,0 +1,21 @@
+linting: bandit black flake8 isort
+	
+testing: coveralls pytest
```

Comment:
I think you will need to run tests before coveralls in order for coveralls to work. And you will need to install pytest-cov in order for a coverage report to be generated (to send to coveralls). What I would do is what we have in the hoard repo: `make test` command runs tests (with coverage), `make coveralls` command requires test, and then runs coveralls. That way you won't need a separate `testing` command to run both.